Makefile

```diff
@@ -61,13 +61,3 @@ run-docs:
 .PHONY: get-version
 get-version:
 	@echo $(shell cat setup.py | grep "_VERSION =" | egrep -o '[0-9]+\.[0-9]+\.[0-9]+(rc[0-9]+)?')
-
-.PHONY: get-last-released-changelog-entry
-get-last-released-changelog-entry:
-	@cat changelogs/$(PKG_VERSION).md
-
-.PHONY: github-action-version-and-changelog
-github-action-version-and-changelog:
-	echo $(PKG_VERSION) > $(HOME)/name
-	echo $(PKG_VERSION) > $(HOME)/tag
-	@cp changelogs/$(PKG_VERSION).md $(HOME)/changelog
```

Comment:
hum, why you're dropping this ?

Reply:
We used to use this task [for the GitHub release](https://github.com/tartiflette/tartiflette/blob/2bb85f7799b5579f7001e1f061dc09002c7c9d2c/.github/workflows/push_on_master.yml#L66), but now this is [directly done](https://github.com/tartiflette/tartiflette/blob/6a283d5f0796ffa74bfe8b794eca27ca19e12901/.github/workflows/push_on_master.yml#L97-L100) in the action that creates the release in the workflow, so the task is not used anymore.

Reply:
:ok_hand: